org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel/src/org/eclipse/xtext/example/domainmodel/serializer/DomainmodelSyntacticSequencer.xtend

```diff
@@ -3,6 +3,16 @@
  */
 package org.eclipse.xtext.example.domainmodel.serializer
 
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.RuleCall
+import org.eclipse.xtext.nodemodel.INode
 
 class DomainmodelSyntacticSequencer extends AbstractDomainmodelSyntacticSequencer {
+	
+	override protected String getArrayBracketsToken(EObject semanticObject, RuleCall ruleCall, INode node) {
```

Comment:
why this change here?

Reply:
(1) this is an older change that is due regeneration with the current mast.
(2) this change leads to [ ] be the default cause there are two keywords in the grammar
(3) this corrects this back

Reply:
OK